### PR TITLE
Updated README.md with example projects

### PR DESCRIPTION
**Pull request by sarmadsangi**
Added following projects for now with a note that people can send PR to add more interesting projects here,

## Examples

* [React Boilerplate](http://reactboilerplate.com) ([source](https://github.com/mxstbr/react-boilerplate))
* [Offline Kanban](https://offline-kanban.herokuapp.com) ([source](https://github.com/sarmadsangi/offline-kanban))
* [Angular CLI](https://github.com/angular/angular-cli) ([source](https://github.com/angular/angular-cli))
* [Phenomic](https://phenomic.io) ([source](https://github.com/MoOx/phenomic))

These are just few projects that are using `offline-plugin`. Feel free to add more such projects here by simply sending a Pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,6 +185,16 @@ Event called when `upUpdating` phase failed by some reason. Nothing is downloade
...
 
 
+## Examples
+
+* [React Boilerplate](http://reactboilerplate.com) ([source](https://github.com/mxstbr/react-boilerplate))
+* [Offline Kanban](https://offline-kanban.herokuapp.com) ([source](https://github.com/sarmadsangi/offline-kanban))
+* [Angular CLI](https://github.com/angular/angular-cli) ([source](https://github.com/angular/angular-cli))
+* [Phenomic](https://phenomic.io) ([source](https://github.com/MoOx/phenomic))
+
+These are just few projects that are using `offline-plugin`. Feel free to add more such projects here by simply sending a Pull request.
```
Comment (NekR) [inline README.md]: I think this must be "If you are using `offline-plugin`, feel free to submit a PR to add your project to this list". This sounds a bit more better, I think

**Comment by NekR** [comment]
Thanks @sarmadsangi! Looks great, thank you for submitting it. I'm going to merge it once you'll fix that sentence.

**Comment by NekR** [comment]
@kentcdodds I think you are using `offline-plugin` too. In which project you are doing it?

**Comment by kentcdodds** [comment]
I'm not actually using it in a real project. Just demoed it in my [es6-todomvc](https://github.com/kentcdodds/es6-todomvc) workshop repo.

**Comment by NekR** [comment]
I see, no problem, thanks for answering :-)

File: README.md (hunk 1 above)
Comment (sarmadsangi) [inline README.md]: Yup this one is better.
done :)

**Comment by NekR** [comment]
@sarmadsangi Okay, merged, thank you for doing this 👍

**Pull request merged**